examples/others/gridnav/lv_example_gridnav_3.py

```diff
@@ -0,0 +1,84 @@
+def cont_sub_event_cb(e):
+    k = e.get_key()
+    obj = e.get_current_target()
+    if k == lv.KEY.ENTER:
+        lv.group_focus_obj(obj)
+
+    elif k == lv.KEY.ESC:
+        lv.grout_t.__cast__(obj.get_group()).focus_next()
```

Comment:
@kisvegabor - What is the reason `lv_obj_get_group` returns `void *` and not `lv_group_t *`?

https://github.com/lvgl/lvgl/blob/965552e4465772dd3c12892216412fc01e651f6e/src/core/lv_obj.h#L314

If it returned `lv_group_t *`, the cast here would not be needed. (type casts are very unintuitive in Python code)

Reply:
Good question. IIRC the were some circular include problems earlier, however it already works. Updated in 61475e5a5

Reply:
Great!

Uli, with https://github.com/lvgl/lvgl/commit/61475e5a5f4aa0d646ab707d6a3e5a57042d8288 now on LVGL master, I think you can remove the `__cast__` here as well.

Reply:
done